util/create_wallet.js

```diff
@@ -0,0 +1,37 @@
+const fs = require('fs')
+const BCHJS = require('@chris.troutner/bch-js')
+const bchjs = new BCHJS()
+
+const COUNT = 10
+
+async function generateTestWallet (addressesCount) {
```

Comment:
I appreciate the work but I think we don't really need extra libraries to build wallets for testing signatures. Hard coded values of signature and texts takes less time for tests to run and are immutable without exposing anything critical.